utils/luci-app-owm/luasrc/owm.lua

```diff
@@ -205,6 +205,15 @@ function showmac(mac)
 	return mac
 end
 
+function get_position()
+	local position = {}
+	uci:foreach("system", "system", function(s) --owm
```

Comment:
think that's the reason
> system.@system[0]=system
system.@system[0].cronloglevel='0'
system.@system[0].zonename='Europe/Berlin'
system.@system[0].timezone='CET-1CEST,M3.5.0,M10.5.0/3'
system.@system[0].hostname='Ahof-frieden03'
system.@system[0].version='0.2.0-alpha'
system.@system[0].location='test'
system.@system[0].latitude='52.436757800213844'
system.@system[0].longitude='13.546518087387085'